src/app/components/MediaLoader/index.tsx

```diff
@@ -298,6 +292,7 @@ const MediaLoader = ({ blocks, className, embedded, uniqueId }: Props) => {
         )}
         {captionBlock && (
           <Caption
+            className={isPortraitVideo ? 'portrait-caption' : ''}
             block={captionBlock}
             type={mediaType}
             css={orientation === 'portrait' && styles.captionPortrait}
```

Comment:
Let's change this here too
```suggestion
            css={isPortraitVideo && styles.captionPortrait}
```